airflow/models/baseoperator.py

```diff
@@ -862,6 +865,7 @@ def clear(self,
             tasks += [
                 t.task_id for t in self.get_flat_relatives(upstream=False)]
 
+        # noinspection PyUnresolvedReferences
```

Comment:
What is the Unresolved reference?

Reply:
It's the in_ operator on task_id. I will try to find another way of excluding those SQLAlchemy magic operators as this is a problem that repeats often. For now I remove it.